hack/local-up-cluster.sh

```diff
@@ -103,7 +103,11 @@ export KUBE_CACHE_MUTATION_DETECTOR
 KUBE_PANIC_WATCH_DECODE_ERROR="${KUBE_PANIC_WATCH_DECODE_ERROR:-true}"
 export KUBE_PANIC_WATCH_DECODE_ERROR
 
-ENABLE_ADMISSION_PLUGINS=${ENABLE_ADMISSION_PLUGINS:-""}
+# Default list of admission Controllers to invoke prior to persisting objects in cluster
+# The order defined here does not matter.
+DEFAULT_ENABLE_ADMISSION_PLUGINS="LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota,StorageObjectInUseProtection"
```

Comment:
the default on plugins are defined in:[https://github.com/kubernetes/kubernetes/blob/72dd54ec4856ccb7bd2b08258d95f3d0b932fb11/pkg/kubeapiserver/options/plugins.go#L132-L142](https://github.com/kubernetes/kubernetes/blob/72dd54ec4856ccb7bd2b08258d95f3d0b932fb11/pkg/kubeapiserver/options/plugins.go#L132-L142)

so we can trim the list, amiright?

Reply:
The only plugin that is in the list but not in the default on plugins list is `StorageObjectInUseProtection` which according to the [cli feature gates docs](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) is enabled by default. 

Regardless of the above, as a user using local-up-cluster.sh I would expect that the default plugins are loaded unless I specify additional plugins using ENABLE_ADMISSION_PLUGINS (or disable plugins using DISABLE_ADMISSION_PLUGINS)

WDYT?